source/codegen/templates/service.cpp.mako

```diff
@@ -13,6 +13,7 @@ custom_types = common_helpers.get_custom_types(config)
 has_async_functions = any(service_helpers.get_async_functions(functions))
 has_two_dimension_functions = any(service_helpers.get_functions_with_two_dimension_param(functions))
 function_names = service_helpers.filter_proto_rpc_functions_to_generate(functions)
```

Comment:
Rename `function_names` to `functions_to_generate` to indicate what this list really contains. After that, other two places below where you are calling `filter_proto_rpc_functions_to_generate`, you can just iterate over `functions_to_generate`. One of these is admittedly existing usage, but seems unnecessary.

Reply:
Updated